pkgs/unified_analytics/lib/src/analytics.dart

```diff
@@ -627,6 +627,50 @@ class AnalyticsImpl implements Analytics {
   }
 }
 
+class FakeAnalytics extends AnalyticsImpl {
+  /// Use this list to check for events that have been emitted when
+  /// invoking the send method
+  final List<Event> sentEvents = [];
+
```

Comment:
@bkonyi thoughts on this so far. I figured that for the fake instance, all you really need is to have access to a list that will contain each `Event` we send so we only really need to override the send method to add to the list

I think all other functionality within the default implementation class should be fine?

Reply:
Yeah, I think this looks good to me! Assuming each `Event` implementation has `operator ==` and `hashCode` overrides, this will work perfectly :).

Reply:
Ah good call!